connexion/validators/json.py

```diff
@@ -68,7 +68,7 @@ def _validate(self, body: t.Any) -> t.Optional[dict]:
             return self._validator.validate(body)
         except ValidationError as exception:
             error_path_msg = format_error_with_path(exception=exception)
-            logger.error(
+            logger.info(
```

Comment:
It would be nice to configure this, as I'd quite like it as a warning; but not an error, and it's clear to me from this that many people may have different use-cases. I Know I can redirect a log-stream.